### ISSUE TITLE
Missing norm argument in several numpy interface fft methods

### ISSUE THREAD

**Issue by timseries** [pyFFTW/pyFFTW#152]
In the [pyfftw numpy interfaces module](https://github.com/pyFFTW/pyFFTW/blob/master/pyfftw/interfaces/numpy_fft.py), some of the fft methods do not have a `norm` argument. For instance: 
```
def fft(a, n=None, axis=-1, norm=None, overwrite_input=False,
        planner_effort='FFTW_ESTIMATE', threads=1,
        auto_align_input=True, auto_contiguous=True)
```

has a `norm` argument, whereas 

```
def rfft(a, n=None, axis=-1, overwrite_input=False,
        planner_effort='FFTW_ESTIMATE', threads=1,
        auto_align_input=True, auto_contiguous=True):
```

does not.

The [numpy version](https://docs.scipy.org/doc/numpy/reference/generated/numpy.fft.rfft.html#numpy.fft.rfft) of the`rfft` method, does have a `norm` argument, as do all other numpy `*fft*` methods.

**Comment by hgomersall** [comment]
Oh, so it doesn't. Fancy raising a PR?

**Comment by grlee77** [comment]
I did look into adding these missing ones I while back, but never finished making the PR.  I can revisit it at some point to see what is still missing.  If @timseries is interested in working on it, please feel free to cherry-pick those commits and start from there
https://github.com/grlee77/pyFFTW/commits/ortho_v2

I think one issue was that I came across a bug in the 'ortho' normalization for numpy's `rfft` which has been fixed in master, but is not in the 1.12.x releases:
https://github.com/numpy/numpy/pull/8445

**Comment by grlee77** [comment]
I went ahead and made a PR for this in #160.

I also want to add the ability to choose the normalisation when using the FFTW builders, but I split that out of the `ortho_v2` branch above and will make that a separate, future PR.
